tapback/backend.c

```diff
@@ -162,6 +162,37 @@ find_tapdisk(const int minor, tap_list_t *tap)
     return err;
 }
 
+static inline int
+find_tapdisk_by_pid(const pid_t pid, const int minor, tap_list_t *tap)
+{
+	struct list_head list = LIST_HEAD_INIT(list);
+	tap_list_t *entry;
+	int err;
+
+	err = tap_ctl_list_pid(pid, &list);
+	if (err) {
+		WARN(NULL, "error listing tapdisks: %s\n", strerror(-err));
+		goto out;
+	}
+
+	err = -ESRCH;
+	if (!list_empty(&list)) {
+		tap_list_for_each_entry(entry, &list) {
+			if (entry->minor == minor && entry->pid == pid) {
+				err = 0;
+				memcpy(tap, entry, sizeof(*tap));
+				break;
+			}
+		}
+        	tap_ctl_list_free(&list);
+        } else {
+		DBG(NULL, "no tapdisks\n");
+	}
```

Comment:
Indentation looks odd?

Reply:
Yeah, it's mixed spaces and tabs again.

Reply:
I don't know how you do this in Vim but I have this for my emacs config
```
(defun linux-c-mode ()
  "C mode for blktap"
  (interactive)
  (c-mode)
  (c-set-style "linux")
  (setq indent-tabs-mode t)
  (setq tab-width 8))

(add-to-list 'auto-mode-alist '(".*blktap.*\\.[ch]$" . linux-c-mode))
```
so that any C/H files under a blktap directory get Linux C mode and display in tabs at 8 chars

Reply:
Fixed, im trying to find an equivalent plugin for vim.